src/test/java/org/broadinstitute/hellbender/JBWAIntegrationTest.java

```diff
@@ -54,4 +57,45 @@ public void testJBWAAlignSingleRead() throws Exception {
         Assert.assertEquals(align[0].getNm(), 0);
         bwaMem.dispose();
```

Comment:
Again, it seems that `index.close()` is necessary here. Actually, does it make sense to have a static `BwaIndex` as field and have something similar to ["@AfterClass"](http://testng.org/javadocs/org/testng/annotations/AfterClass.html)?